ts/data.ts

```diff
@@ -1,6 +1,33 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+interface Log {
+  title: string;
+  url: string;
+  notes: string;
+  id: number;
+}
+let tempID: number = 1;
+const entryArray: Log[] = readLogs();
 const data = {
   view: 'entry-form',
-  entries: [],
+  entries: entryArray,
   editing: null,
-  nextEntryId: 1,
+  nextEntryId: tempID,
 };
+console.log(JSON.parse(JSON.stringify(data)));
+
+function readLogs(): Log[] {
+  if (localStorage.getItem('logs')) {
+    const newLogs: Log[] = JSON.parse(localStorage.getItem('logs') as string);
+    tempID = Number(JSON.parse(localStorage.getItem('id') as string));
+    return newLogs;
+  } else {
+    return [];
+  }
+}
+
+function writeLogs(): void {
```

Comment:
We need to send the `data` object to local storage. There shouldn't be a need to send two different things to local storage. After you do that there may be some refactoring you need to do to access your values in local storage.

This also means that you will need to rewrite your interface for the `Data` object instead of `Log`

Someting like: 
```
interface Data {
  view: 'entries' | 'entry-form';
  entries: Log[];
  editing: null | Entry;
  nextEntryId: number;
}
```